action.yml

```diff
@@ -3,20 +3,67 @@ name: 'Composite Run Steps Action Template'
 description: 'Add description here'
 
 inputs:
-  input-1:
-    description: 'Add a description here'
+  db-server-name:
+    description: The database server name.
     required: true
+  db-server-port:
+    description: The port the database server listens on.
+    required: false
+    default: '1433'
+  db-name:
+    description: The name of the database to run tests against.
+    required: true
+  query-timeout:
+    description: An optional setting, in seconds, for when to timeout the query that runs the tests. If tests sometimes hang, or they shouldn't take longer than a certain amount of time, this parameter can be helpful.
+    required: false
+  use-integrated-security:
+    description: A switch defining whether or not to use integrated security. If not provided, a password should be.
+    required: true
+    default: 'false'
+  db-username:
+    description: The username to use to login to the database.
+    required: false
+  db-password:
+    description: The password for the user logging in to the database.
+    required: false
```

Comment:
Maybe the description for these two should say they aren't required if using the integrated security option?